Library/Homebrew/keg.rb

```diff
@@ -105,7 +105,13 @@ def self.find_some_installed_dependents(kegs)
     #
     # This happens after the initial dependency check because it's sloooow.
     remaining_formulae = Formula.installed.select do |f|
-      f.installed_kegs.any? { |k| Tab.for_keg(k).runtime_dependencies.nil? }
+      installed_kegs = f.installed_kegs
+
+      # All installed kegs are going to be removed anyway,
```

Comment:
Can you elaborate a bit more on this? The "removed anyway" bit makes sense in context of the PR but I'm not sure it does in the context of the function itself which isn't necessarily relating specifically to `uninstall`.